src/main/java/nextstep/courses/domain/Sessions.java

```diff
@@ -0,0 +1,16 @@
+package nextstep.courses.domain;
+
+import java.util.List;
+
+public class Sessions {
+
+    private List<Session> sessions;
+
+    public Sessions() {
+    }
```

Comment:
```suggestion
    private final List<Session> sessions;

    public Sessions() {
        sessions = new ArrayList<>();
    }
```
이렇게 보완이 되어야할 것 같네요 😄